src/transformers/models/hubert/modeling_tf_hubert.py

```diff
@@ -227,7 +227,7 @@ def _compute_mask_indices(
             f" `sequence_length`: {sequence_length}`"
         )
     # compute number of masked spans in batch
-    num_masked_spans = int(mask_prob * sequence_length / mask_length + tf.random.uniform((1,)))
+    num_masked_spans = int(mask_prob * sequence_length / mask_length + np.random.uniform(size=(1,)))
```

Comment:
Using `tf.random.uniform` here creates issues with int casting here, `max` on L231 and other logic that assumes an int downstream.

Reply:
@amyeroberts won't using `np` create problems in graph mode? For e.g. if the model was compiled?

Reply:
Yep. I've removed it and added all the necessary tf logic.